Integrations/integration-Pwned.yml

```diff
@@ -67,38 +68,66 @@ script:
         return md;
     }
 
+    var dataToEc = function(type,arg,obj) {
+        var ec = {};
+        if (obj && obj.length>0) {
+            ec.Pwned = {'Type':type,'Data':arg};
+            if (type == 'email') {
+                ec.Pwned.Domains = [];
+                for (var i=0;i<obj.length;i++) {
+                    ec.Pwned.Domains.push(obj[i].Domain);
```

Comment:
I think it should be the other way around.
Domain.Pwned
Since we have a domain object already.
And then you can state on it whether it was pwned or not